rwengine/src/objects/VehicleObject.cpp

```diff
@@ -166,7 +202,10 @@ void VehicleObject::tickPhysics(float dt)
 		// todo: a real engine function
 		float velFac = info->handling.maxVelocity;
 		float engineForce = info->handling.acceleration * throttle * velFac;
-		if( fabs(engineForce) >= 0.001f ) physBody->activate(true);
+		if( fabs(engineForce) >= 0.001f )
+		{
```

Comment:
nit: style wise another instance in this file uses a single line for `if ( condition ) {`. this one has a newline before the brace